File: rotate_image.java
public class rotate_image {
	public static void main(String args[]) { 
		// run tests here
	}

	public static int[][] rotateImage(int[][] a) {
		int size = a.length; 
		int[][] result = new int[size][size];
		
		for(int row = 0; row < size; row++){
			int new_row = size - row - 1;
			for(int column = 0; column < size; column++){
				// System.out.println("Inserted: " + a[row][column] + " into " + " (" + column + ", " + new_row + ")");
				result[column][new_row] = a[row][column];
			}
		}
		
		return result;
	}
}

File: rotate_image.py
def rotateImage(a):
    b = a # create a clone of a
    size = len(a)
    
    for row, columnS in enumerate(a): # row = index, columnS = list object
        new_row = size - row - 1
        # print(columnS)
        for column, num in enumerate(columnS): # column = index, num = VALUE
            # print(num)
            print(str(num) + " inserted into: (" + str(column) + ", " + str(new_row) + ")")
            # b[column][new_row] = num
        # print("\n")
        
    return b

a = [[1, 2, 3],
     [4, 5, 6],
     [7, 8, 9]]

b = rotateImage(a)
for i in b:
	print(i)